core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -3803,6 +3803,33 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-ibmcloud
     namespace: ci
+- from:
+    pull-secret:
+      dockerconfigJSON:
+      - auth_field: token_image-puller_app.ci_reg_auth_value.txt
+        item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_field: auth
+        email_field: email
+        item: cloud.openshift.com-pull-secret
+        registry_url: cloud.openshift.com
+      - auth_field: auth
+        email_field: email
+        item: quay.io-pull-secret
+        registry_url: quay.io
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+  to:
+  - cluster_groups:
```

Comment:
I think you can append the `cluster-secrets-ibmcloud-qe` item to https://github.com/openshift/release/blob/fdc50e4098e77554234c432288cf915bdef77042/core-services/ci-secret-bootstrap/_config.yaml#L3801